src/main/java/frc/robot/RobotContainer.java

```diff
@@ -19,45 +20,53 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
-  // Replace with CommandPS4Controller or CommandJoystick if needed
-  private final CommandXboxController m_driverController =
-      new CommandXboxController(OperatorConstants.kDriverControllerPort);
+  /* Setting up bindings for necessary control of the swerve drive platform */
 
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
-  public RobotContainer() {
-    // Configure the trigger bindings
-    configureBindings();
+  private final CommandPS4Controller joystick = new CommandPS4Controller(0);
```

Comment:
This port should be a constant or something

Reply:
I agree